monitoring/grafana/README.md

```diff
@@ -4,6 +4,17 @@ that will help you get started quickly introducing monitoring capabilities to yo
 
 Quick Start
 -----------
+Before generating the dashboards on your own, note that you may find the sample

```

Comment:
I'm not a native English speaker, so not sure about this one, but: would it be better to use "can" instead of "may" here? I.e. "you can find" instead of "you may find".

Reply:
I'm not a native English speaker as well, but it seems that [you're right](https://writingexplained.org/can-vs-may-difference). 😄